netlify.toml

```diff
@@ -14,3 +14,15 @@ targetPort = 3001
 
 [functions]
 deno_import_map = "./netlify/edge-functions/deno.json"
+
+[[redirects]]
+  from = "https://insight.opensauced.pizza/*"
+  to = "https://insights.opensauced.pizza/:splat"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "https://app.opensauced.pizza/*"
+  to = "https://insights.opensauced.pizza/:splat"
+  status = 307
```

Comment:
I probably don't have the full context as to why, but any reason this one is a temporary redirect?

Reply:
I made it temporary in the case that we flip them in the future, where `app` is the primary, and `insights` would then permanently redirect to `app`